lib/StaticNarrative/exporter/preprocessor.py

```diff
@@ -6,18 +6,23 @@
 import os
 from collections import defaultdict
 from datetime import datetime
+from typing import T
 
 from installed_clients.NarrativeMethodStoreClient import NarrativeMethodStore
 from nbconvert.preprocessors import Preprocessor
+from nbformat import NotebookNode
+
 from StaticNarrative.upa import deserialize
 
 from .app_processor import AppProcessor
 from .processor_util import get_authors, get_icon
 
 
 class NarrativePreprocessor(Preprocessor):
-    def __init__(self, config=None, **kw):
-        super(NarrativePreprocessor, self).__init__(config=config, **kw)
+    def __init__(
+        self: "NarrativePreprocessor", config: T | None = None, **kw: dict[str, T]
+    ) -> None:
+        super().__init__(config=config, **kw)
```

Comment:
simpler form of calling `super()`

(calling `super()` also fixes the check failure above)